rust/query/match_clause.rs

```diff
@@ -46,10 +46,6 @@ impl MatchClause {
         Self { conjunction }
     }
 
-    pub fn from_patterns(patterns: Vec<Pattern>) -> Self {
-        Self::new(Conjunction::new(patterns))
-    }
```

Comment:
This was only used in the builder and is easily inlined by simply repeating the body of this function.